src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.cpp

```diff
@@ -164,15 +163,18 @@ bool FixConservatives::operator()(
             : 0.;
 
     // Equation B.40 of Foucart
-    const double lower_bound_of_lorentz_factor =
-        std::max(1. + tau_over_d - b_squared_over_d, 1.);
+    const double lower_bound_of_lorentz_factor_minus_one =
+        std::max(tau_over_d - b_squared_over_d, 0.);
     // Equation B.31 of Foucart evaluated at lower bound of lorentz factor
     const double simple_upper_bound_for_s_tilde_squared =
-        square(lower_bound_of_lorentz_factor + b_squared_over_d) *
-        (square(lower_bound_of_lorentz_factor) - 1.) /
-        (square(lower_bound_of_lorentz_factor) +
+        square(1.0 + lower_bound_of_lorentz_factor_minus_one +
```

Comment:
Self change request: Do this same rewrite below where the expression is evaluated with the real Lorentz factor.  Maybe factor it into a lambda.

Reply:
Done and squashed.